src/style.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background-color: #fff;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: left center;
+}
+
+.form-big {
+  font-size: 16px;
+  margin: 12px 0;
+  height: 70px;
+  background-position: 25px 24px;
+  padding-left: 62px;
+}
+
+.form-small {
+  font-size: 14px;
+  margin: 8px 0;
+  height: 42px;
+  background-position: 12px 14px;
+  background-size: 1.64%;
+  padding-left: 33px;
```

Comment:
it would be better to add the image using `:before` for label, instead of set it as background for form

you can set `position: relative` to `label`, and then define width, height, and position parameters in order to aline image

for example

```
.label:before {
    content: "";
    position: absolute;
    width: 19px;
    height: 19px;
    left: 26px;
    top: 25px;
    background-image: url('./images/Search.svg');
}
```

This approach more flexible and more easy to manage